index.bs

```diff
@@ -1080,16 +1097,17 @@ Reads back the {{MLTensor/[[data]]}} of an {{MLTensor}} from the {{MLContext}}.{
     1. If |tensor|.{{MLTensor/[[isDestroyed]]}} is true, then return [=a new promise=] [=rejected=] with a {{TypeError}}.
     1. If |tensor|.{{MLTensor/[[descriptor]]}}.{{MLTensorDescriptor/readable}} is false, then return [=a new promise=] [=rejected=] with a {{TypeError}}.
     1. Let |promise| be [=a new promise=].
-    1. Enqueue the following steps to |tensor|.{{MLGraph/[[context]]}}.{{MLContext/[[timeline]]}}:
+    1. Enqueue the following steps to |tensor|.{{MLGraph/[[context]]}}.{{MLContext/[[timeline]]}}, which [=/abort when=] [=this=] [=MLContext/is lost=]:
```

Comment:
Is `|tensor|.{{MLGraph/[[context]]}}` a typo (an old issue)? Should it be `|tensor|.{{MLTensor/[[context]]}}` or just `[=this=].{{MLContext/[[timeline]]}}`? If it is a typo, the line 1096 also needs to be fixed. For example,
```suggestion
    1. Enqueue the following steps to [=this=].{{MLContext/[[timeline]]}}, which [=/abort when=] [=this=] [=MLContext/is lost=]:
```

Reply:
Another question is whether it should check context lost earlier? Like `createTensor` steps do, e.g.

```
1. If [=this=] [=MLContext/is lost=], then return [=a new promise=] [=rejected=] with an "{{InvalidStateError}}" {{DOMException}}.
```

Reply:
Given that the context loss will destroy every associated tensor, I don't think that is needed. I'm not opposed to adding it explicitly though.

Reply:
For the "type confusion" I added a lint tool test and caught several more places. It's not perfect, but it is progress!